### PR TITLE
Fix user sign up

### PR DESCRIPTION
**Pull request by kevdaci**
## Commit Message Guidelines
```
Header
Blank Line
Body
Blank Line
Footer

The header should look like:
<type>(<scope>): <subject>

The body should have any necessary detailed info about the commit:
An example, references as to where this idea came from, etc.

The footer should have all the issues tagged:
Fixes #123, Fixes #456

So a commit should look like:
feat(users): Add new Yahoo authentication

Yahoo authentication idea proposed by @codydaig
Example implementation in file.js

Fixes #82
```

* Types: 
  * feat - Features, Enhancements, and overall Improvements
  * fix - Fixes, Bugs, HotFixs, etc...
  * doc - Changes to the Documentation and doesn't actually touch any code.
* Scope:
  * The scope should be where the change took place.
  * Examples: users, core, config, articles
* Subject:
  * The subject line should be clear and concise as to what is being accomplished in the commit.
* General Rules:
  * No Line in the Commit message can be longer than 80 characters.
* Reference: [Angular Conventions](https://github.com/ajoslin/conventional-changelog/blob/master/conventions/angular.md)

### CODE REVIEW

**Comment by markd315** [comment]
I didn't do a good job of communicating this, but we don't need the user to sign up for a role anymore if we're just letting the admin assign it. It would make some sense to leave his request in as a default but why bother when we can just hide this role info from the user entirely and leave it to Dmitry.

File: modules/users/client/views/authentication/signup.client.view.html [hunk 1]
```diff
@@ -26,6 +26,17 @@ <h3 class="col-xs-12 text-center">Sign up</h3>
           </div>
         </div>
         <div class="form-group" show-errors>
+          <label for="roles">Role</label>
```
Comment (markd315) [inline modules/users/client/views/authentication/signup.client.view.html:29]: Scratch this, we'll just do it from admin/addnewuser and admin/approval

**Review by markd315 — CHANGES_REQUESTED** [review]
Just take out some of this functionality from the signup branch.

File: modules/users/client/views/authentication/signup.client.view.html (hunk 1 above)
Comment (kevdaci) [reply modules/users/client/views/authentication/signup.client.view.html:29]: Would it be convenient for the admin to see what role the user is applying for? I believe that the admin would have a hard time to search up the role for every user that is applying to the system. I feel like adding this feature would make life easier for the admin.

Comment (markd315) [reply modules/users/client/views/authentication/signup.client.view.html:29]: I don't really think it's a concern worth spending time on, and my real reason is that I had a good deal of trouble with harnessing "default" roles on the approval view so the work would go to waste anyway. It's not part of a story and we don't need the functionality.

**Pull request closed without merging**